zk/l1infotree/updater_internal_test.go

```diff
@@ -0,0 +1,114 @@
+package l1infotree
+
+import (
+	"context"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/ledgerwatch/erigon-lib/common"
+	"github.com/ledgerwatch/erigon-lib/kv"
+	"github.com/ledgerwatch/erigon-lib/kv/mdbx"
+	"github.com/ledgerwatch/erigon/core/types"
+	"github.com/ledgerwatch/erigon/zk/hermez_db"
+	"github.com/stretchr/testify/assert"
+)
+
+// to be removed
+func GetDbTx() (tx kv.RwTx, cleanup func()) {
+	dbi, err := mdbx.NewTemporaryMdbx(context.Background(), "")
+	if err != nil {
+		panic(err)
+	}
+	tx, err = dbi.BeginRw(context.Background())
+	if err != nil {
+		panic(err)
+	}
+
+	err = hermez_db.CreateHermezBuckets(tx)
+	if err != nil {
+		panic(err)
+	}
+
+	return tx, func() {
+		tx.Rollback()
+		dbi.Close()
+	}
+}
+
+func TestUpdater_chunkLogs(t *testing.T) {
+
+	logs := []types.Log{
+		{
+			Address: common.Address{},
+			Topics:  []common.Hash{},
+			Data:    []byte{},
+		},
+	}
+
+	chunkedLogs := chunkLogs(logs, 1)
+	assert.Len(t, chunkedLogs, 1)
```

Comment:
Is it worth having say 4 logs here and chunking it into blocks of 2 for a more extensive check?

Reply:
will do this one with ease.